docs/dev/project_management.rst

```diff
@@ -8,16 +8,52 @@ management tools as well as `Zenhub <https://www.zenhub.com>`__ which provides
 additional features for sprint planning, task estimation, and progress reports.
 
 -------------------------------------------------------------------------------
-Issues and Project Tracking
+Agile Scrum Kanban Iteration SDLC Process
 -------------------------------------------------------------------------------
-We use `Github issues <https://github.com/catalyst-cooperative/pudl/issues>`__ to
-track bugs, enhancements, support requests, and just about any other work that goes
-into the project. Try to make sure that issues have informative tags so we can find
-them easily.
-
-We use Zenhub Sprints, Epics, and Releases to track our progress. These won't be
-visible unless you have the `ZenHub browser extension
-<https://www.zenhub.com/extension>`__ installed.
+
+We merge code to ``dev`` continuously, but plan work in two-week sprints.
+
+We track work as items (GitHub issues and PRs) in a Catalyst-wide Github Project.
+
+During each sprint, people on each team pull from a team-specific list of
+prioritized & scoped items.
+
+We have 3 meetings per sprint:
+
+- planning meeting: overall goal is "get a good idea of what work is coming up
+  in the sprint"
+
+  - mark items with a new iteration
+
+    - enough that that we feel like there's enough work to do
+
+    - few enough that we don't feel overwhelmed
+
+- mid-sprint check-in: overall goal is "given this sprint's status, prioritize
+  items for the next sprint"
+
+  - make sure item status is up-to-date
+
+  - identify items that need clarification / re-scoping
+
+  - identify items that we'd like to pull into the backlog
```

Comment:
The mid sprint checkin doesn't have tons of structure right now. We do usually hit the first two bullets - do we want to start doing some sort of prioritization? We don't have to - this is just a suggestion.